### PR TITLE
Set up linting and test stub for notebook Python client code.

### PR DESCRIPTION
**Pull request by markfickett**

### CODE REVIEW

File: client/py/module_test.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import unittest
+import aou_workbench_client  # pylint: disable=unused-import
+from aou_workbench_client import swagger_client  # pylint: disable=unused-import
+
+
+class ModuleTest(unittest.TestCase):
+
+  def test_noop_as_verification_that_module_imports_worked(self):
```
[the remark this answers is not in the document]
Comment (markfickett) [reply client/py/module_test.py]: We do have indentation checking -- in `pylintrc` it's `indent-string='  '`. I went with two spaces for this project to match RDR and Google-internal Python style. I see the external guide also says 4 spaces (https://google.github.io/styleguide/pyguide.html#Indentation), though I think it doesn't differentiate normal indents and wrapped lines.

I'm used to / like the look of 2 spaces, but don't mind switching. I do feel strongly that wrapped lines should be visibly different, like:
```Python
if True:
  x = 1
  y = some_very_long_function_call(
      indent_its_wrapped_arg_list_twice_usual,
      so_that_the_arg_list_doesnt_look_like,
      a_normal_indented_code_block)
```

[the remark this answers is not in the document]
Comment (blrubenstein) [reply client/py/module_test.py]: I think I also personally prefer two spaces, but this is also something that we could discuss at the 2:00 meeting.

File: libproject/pylintrc [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+
+# Template used to display messages. This is a python new-style format string
+# used to format the message information. See doc for all details
+#msg-template=
+
+
+[BASIC]
+
+# List of builtins function names that should not be used, separated by a comma
+bad-functions=map,filter,apply,input
```
Comment (markfickett) [inline libproject/pylintrc]: rm these

Comment (blrubenstein) [inline libproject/pylintrc]: Probably don't want to forbid functions, just prefer list comprehensions when we can.

**Comment by markfickett** [comment]
> changing the run commands for both build_and_test_and_deploy_api and build_and_test_and_deploy_ui
Just removing an extra line wrap (in the spirit of brevity).

> discuss linting/styling for python
Done, yay!

> don't want to forbid functions
Removed.

> standard in the Python world (via PEP 8) is 4 spaces
Will follow up with that change to keep diffs sane.

**Pull request merged**